storage/database/src/main/java/de/danoeh/antennapod/storage/database/PodDBAdapter.java

```diff
@@ -1046,13 +1046,13 @@ public void setFeedItems(int oldState, int newState, long feedId) {
      * Excludes those feeds that do not have 'Keep Updated' enabled.
      * The returned cursor uses the FEEDITEM_SEL_FI_SMALL selection.
      */
-    public final Cursor getNewItemsCursor(int offset, int limit) {
+    public final Cursor getNewItemsCursor(int offset, int limit, String sortType) {
         final String query = SELECT_FEED_ITEMS_AND_MEDIA
                 + " INNER JOIN " + TABLE_NAME_FEEDS
                 + " ON " + TABLE_NAME_FEED_ITEMS + "." + KEY_FEED + "=" + TABLE_NAME_FEEDS + "." + KEY_ID
                 + " WHERE " + TABLE_NAME_FEED_ITEMS + "." + KEY_READ + "=" + FeedItem.NEW
                     + " AND " + TABLE_NAME_FEEDS + "." + KEY_KEEP_UPDATED + " > 0"
-                + " ORDER BY " + TABLE_NAME_FEED_ITEMS + "." + KEY_PUBDATE + " DESC"
+                + " ORDER BY " + TABLE_NAME_FEED_ITEMS + "." + KEY_PUBDATE + " " + sortType
```

Comment:
If you pass an enum as parameter, I think it would be cool if the ordering therm could be managed by its own class (so it can be reused on other screens later). I'm thinking about something like the `FeedItemFilterQuery` class, just for sort orders.